packages/clients/meldemichel/package.json

```diff
@@ -38,5 +39,8 @@
     "@polar/plugin-scale": "^1.0.0",
     "@polar/plugin-toast": "^1.0.0",
     "@polar/plugin-zoom": "^1.0.0"
+  },
+  "peerDependencies": {
+    "@masterportal/masterportalapi": "2.8.0"
```

Comment:
Why is it needed as a peerDependency?

Reply:
https://github.com/Dataport/polar/pull/42/commits/7a19d98125c12b11cbbc1785b5db3403f4dd5ea9

I guess I moved the code requiring it to the core later, but don't remember.